src/main/java/hudson/plugins/jira/JiraBuildAction.java

```diff
@@ -11,11 +15,12 @@
  * @author Kohsuke Kawaguchi
  */
 public class JiraBuildAction implements Action {
-    public final AbstractBuild<?, ?> owner;
+    
+	public final Run<?, ?> owner;
```

Comment:
I know of no plugin that currently depends on jira plugin. Is anywhere documentation of this dependencies or some other way to find it? If someone have plugin that depends on jira plugin, please point me and i check if this change is problem.